doc/classes/GDExtension.xml

```diff
@@ -1,49 +1,64 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GDExtension" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A shared native library for GDExtension.
 	</brief_description>
 	<description>
+		The [GDExtension] resource type represents a set of libraries written for GDExtension. These libraries expand the existing functionality of the engine. They are handled by the [GDExtensionManager] singleton.
+		[b]Note:[/b] Do not worry about GDExtension unless you know what you are doing.
+		[b]Note:[/b] GDExtension itself is not a scripting language and has no relation to [GDScript] resources.
 	</description>
 	<tutorials>
+		<link title="GDExtension overview">$DOCS_URL/tutorials/scripting/gdextension/what_is_gdextension.html</link>
+		<link title="GDExtension example in C++">$DOCS_URL/tutorials/scripting/gdextension/gdextension_cpp_example.html</link>
 	</tutorials>
 	<methods>
 		<method name="close_library">
 			<return type="void" />
 			<description>
+				Closes the current library.
 			</description>
 		</method>
 		<method name="get_minimum_library_initialization_level" qualifiers="const">
 			<return type="int" enum="GDExtension.InitializationLevel" />
 			<description>
+				Returns the lowest level required for this extension to be properly initialized. Ideally, if this value is lower than the current level, this [GDExtension] should be restarted to ensure it is properly applied.
```

Comment:
I don't know that we need the extra detail here:

```suggestion
				Returns the lowest level required for this extension to be properly initialized.
```

Reply:
Unfortunately, the extra detail may need to be mentioned somehow, since the user can use this method combined with the others about library initialization are exposed.